src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -350,70 +392,67 @@
          the package is used, it will reload the source-built version -->
     <ItemGroup>
-      <_FilesToCopy Include="$(PackagesDir)$([System.String]::copy('%(_BuiltPackageInfos.PackageId)').ToLower())/%(_BuiltPackageInfos.PackageVersion)/**/*.nupkg" />
-      <_FilesToDelete Include="$(PackagesDir)$([System.String]::copy('%(_BuiltPackageInfos.PackageId)').ToLower())/%(_BuiltPackageInfos.PackageVersion)/**/*.*" />
+      <_FilesToCopy Include="$(NuGetPackageRoot)$([System.String]::copy('%(_BuiltPackageInfos.PackageId)').ToLowerInvariant())/%(_BuiltPackageInfos.PackageVersion)/**/*.nupkg" />
+      <_FilesToDelete Include="$(NuGetPackageRoot)$([System.String]::copy('%(_BuiltPackageInfos.PackageId)').ToLowerInvariant())/%(_BuiltPackageInfos.PackageVersion)/**/*.*" />
     </ItemGroup>
+
     <Copy SourceFiles="@(_FilesToCopy)" DestinationFolder="$(PreviouslyRestoredPackagesPath)$(RepositoryName)/" />
     <Delete Files="@(_FilesToDelete)" />
 
-    <WriteLinesToFile File="$(RepoCompletedSemaphorePath)RemoveBuiltPackagesFromCache.complete" Overwrite="true" />
+    <MakeDir Directories="$(BaseIntermediateOutputPath)" />
+    <Touch Files="$(BaseIntermediateOutputPath)RemoveBuiltPackagesFromCache.complete" AlwaysCreate="true">
+      <Output TaskParameter="TouchedFiles" ItemName="FileWrites" />
+    </Touch>
   </Target>
 
-  <Target Name="DisplayDirSizeBeforeBuild"
-          BeforeTargets="Build"
-          Condition=" '$(CleanWhileBuilding)' == 'true' ">
-    <Message Text="DirSize Before Building $(RepositoryName)" Importance="High" />
-    <Exec Command="df -h $(ProjectDir)" />
-  </Target>
-  <Target Name="DisplayDirSizeAfterBuild"
-          AfterTargets="Build"
-          BeforeTargets="CleanupRepo"
-          Condition=" '$(CleanWhileBuilding)' == 'true' ">
-    <Message Text="DirSize After Building $(RepositoryName)" Importance="High" />
-    <Exec Command="df -h $(ProjectDir)" />
-  </Target>
-  <Target Name="DisplayDirSizeAfterClean"
-          AfterTargets="CleanupRepo"
-          Condition=" '$(CleanWhileBuilding)' == 'true' ">
-    <Message Text="DirSize After CleanupRepo $(RepositoryName)" Importance="High" />
-    <Exec Command="df -h $(ProjectDir)" />
-  </Target>
-
-  <Target Name="CleanupRepo" 
+  <Target Name="CleanupRepo"
           AfterTargets="RemoveBuiltPackagesFromCache"
-          Condition=" '$(CleanWhileBuilding)' == 'true' ">
-
-    <!-- Make a copy of the build logs & project.assets.json files -->
+          Condition="'$(CleanWhileBuilding)' == 'true' and Exists('$(ProjectDirectory)artifacts')">
     <PropertyGroup>
-      <BuildLogsDir>$(ProjectDirectory)artifacts/buildLogs</BuildLogsDir>
-      <BuildObjDir>$(ProjectDirectory)artifacts/buildObj</BuildObjDir>
+      <BuildLogsDir>$([MSBuild]::NormalizeDirectory('$(ProjectDirectory)', 'artifacts', 'buildLogs'))</BuildLogsDir>
+      <BuildObjDir>$([MSBuild]::NormalizeDirectory('$(ProjectDirectory)', 'artifacts', 'buildObj'))</BuildObjDir>
```

Comment:
Why is this not just artifacts/obj and artifacts/log?

Reply:
No idea, this already existed before. I think we can clean this up further after the inner clone change.